src/rules/dollar-variable-no-namespaced-assignment/index.js

```diff
@@ -0,0 +1,31 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("dollar-variable-no-namespaced-assignment");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "Expected $ variable name to match specified pattern"
```

Comment:
Same thing in this rule as in the other one:

I went and checked from the stylelint repo, and most of the `-no-` rules seem to use the `rejected: "Unexpected ..."` message format, should this rule also use it?

Reply:
I fixed it (and the whole message was wrong anyway due to a bad copy-paste)